readme.md

```diff
@@ -66,6 +88,12 @@ Describe the purpose of views and models in FE JS.
 Your Answer:
 ```text
 
+Views and Models in Front End JS
+
+Views - Views are responsible for the layout and what appears on the page when a browser requests it. The view files in OOJS are typically named after their model+View -- in this example you can generate a constructor method that will generate a view of a particular instance of a model.
+
+Models - Models are responsible for culling the information from the back-end databases, and creating the methods which will be used by the views. The Models ask the back-end for the data about the collections and its objects and make it accessible.
+
 ```
```

Comment:
:+1:

Although the models are not entirely responsible for providing the view with methods. In fact, anything that has to do with rendering data or responses to user interaction should be defined in our views.